main.go

```diff
@@ -366,7 +368,11 @@ func (bridge *Bridge) requestStartSync() {
 func (bridge *Bridge) startWebsocket() {
 	onConnect := func() {
 		// TODO disable this for non-mac connectors once they send bridge status updates themselves
-		go bridge.SendBridgeStatus(imessage.BridgeStatus{StateEvent: BridgeStatusConnected})
+		if bridge.latestState != nil {
```

Comment:
Just realized that this might cause the non-cloud platforms to always send the bridge startup timestamp since it caches the first `SendBridgeStatus` data.

The TODO should also be removed, but other than that seems fine